ga4gh/exceptions.py

```diff
@@ -0,0 +1,83 @@
+"""
+Module containing error objects we return to clients
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+class ParentException(Exception):
+
+    def __init__(self):
+        super(ParentException, self).__init__()
+        self.message = ""
+        self.code = ""
+        self.description = ""
+        self.httpStatus = None
+
+    def toDict(self):
+        return {
+            "message": self.message,
+            "code": self.code,
+            "description": self.description,
+        }
+
+
+class BadRequestException(ParentException):
+
+    def __init__(self):
+        super(BadRequestException, self).__init__()
+        self.httpStatus = 400
+        self.message = "Bad request"
+        self.code = "GAE1"
```

Comment:
Code has to be an integer for the protocol - if we return a string, then we won't be conformant. I agree we'll want to have a systematic way of handling these in the future and just putting in integers isn't a good idea, but it'll get us started and we can figure out how to do it properly later.